2023/input/day23

```diff
@@ -0,0 +1,23 @@
+#.#####################
```

Comment:
Apparently these inputs shouldn't be shared (look it up), so now we will no longer accept them :)

Reply:
I know! These aren't my inputs. They're the example :)

Reply:
well I think even the sample input is still protected